libraries/chain/webassembly/runtimes/eos-vm-oc/executor.cpp

```diff
@@ -74,7 +74,7 @@ static void segv_handler(int sig, siginfo_t* info, void* ctx)  {
 
 static intrinsic grow_memory_intrinsic EOSVMOC_INTRINSIC_INIT_PRIORITY("eosvmoc_internal.grow_memory", IR::FunctionType::get(IR::ResultType::i32,{IR::ValueType::i32,IR::ValueType::i32}),
   (void*)&eos_vm_oc_grow_memory,
-  find_intrinsic_index("eosvmoc_internal.grow_memory")
+  std::integral_constant<std::size_t, find_intrinsic_index("eosvmoc_internal.grow_memory")>::value
```

Comment:
Please change these to `std::integral_constant_v`

Reply:
I don't think there is a `std::integral_constant_v<>`. it is either `::value` as it was instantiate it and it will cast to a `std::size_t`

Reply:
Gotcha.